noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -802,10 +1267,14 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST7">
         <title>File period containment</title>
+        <description>
+            Tests whether the Files' created and finalized dates
```

Comment:
`Tests whether the created and finalized dates of all files are within the archival period specified in arkivuttrekk.xml`